### PR TITLE
changed description

### PR DESCRIPTION
**Pull request by calum-gray**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
     "name": "networkrailbusinesssystems/activity-log",
-    "description": "Ahhhhh",
+    "description": "A library for documenting the activity log of a system",
```
Comment (AnthonyEdmonds) [inline composer.json:3]: Not quite accurate. This library does not document anything. It contains the common activity log components, controllers, views, etc, that support our systems.

**Review by AnthonyEdmonds — CHANGES_REQUESTED** [review]

File: composer.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
     "name": "networkrailbusinesssystems/activity-log",
-    "description": "Ahhhhh",
+    "description": "A library that contains the common activity log components supporting our GovUk systems",
```
Comment (AnthonyEdmonds) [inline composer.json:3]: > A library that contains the common activity log components supporting our GovUk systems

We already know that this is a library, so we can shorten it a little. GOV.UK is the correct way to refer to it.

> Common activity log components for GOV.UK systems

Sounds fine to me.

**Review by AnthonyEdmonds — CHANGES_REQUESTED** [review]

**Review by AnthonyEdmonds — CHANGES_REQUESTED** [review]
Add it to the Readme along with a header, and put it on Github too.

**Review by AnthonyEdmonds — APPROVED** [review]

**Comment by calum-gray** [comment]
> Add it to the Readme along with a header, and put it on Github too.

Added it to readme on a separate pull request, is it okay to close this now

**Pull request merged**